### ISSUE TITLE
Open Selected Log Curve value View on another server (including the interval selected)

### ISSUE THREAD

**Issue by ThomasHalland** [equinor/witsml-explorer#2098]
When I have made a log selection on a server, and opened to see the values, I would like an option to "Open view on server" which will open the exact same curve selection including the interval on the selected server. This shall be opened in a seperate window.

**Comment by eliasbruvik** [comment]
This would require the value view with mnemonics and interval to be a part of the routing. Should be easy to implement once #2114 is done.

**Comment by janmarius** [comment]
Duplicate of #2153
